packages/analytics-browser/src/config.ts

```diff
@@ -213,10 +213,9 @@ export const useBrowserConfig = async (
   amplitudeInstance.previousSessionUserId = previousCookies?.userId ?? legacyCookies.userId;
 
   const trackingOptions = {
-    ...options.trackingOptions,
-    ipAddress: true,
-    language: true,
-    platform: true,
+    ipAddress: options.trackingOptions?.ipAddress === undefined ? true : options.trackingOptions.ipAddress,
```

Comment:
nit: I think `??` could be useful here https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing
```suggestion
    ipAddress: options.trackingOptions?.ipAddress ?? true,
```

Reply:
Take the solution Kevin provided to reverse the order.